plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/preference/CommitterPreferencePresenter.java

```diff
@@ -48,8 +43,12 @@ public CommitterPreferencePresenter(CommitterPreferenceView view,
         this.view = view;
         this.preferencesManager = preferencesManager;
 
-        name = firstNonNull(preferencesManager.getValue(COMMITTER_NAME), DEFAULT_COMMITTER_NAME);
-        email = firstNonNull(preferencesManager.getValue(COMMITTER_EMAIL), DEFAULT_COMMITTER_EMAIL);
+        if (preferencesManager.getValue(COMMITTER_NAME) != null ) {
```

Comment:
Why do you check value on null before storing value into variable which have null value?